pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -130,3 +142,8 @@ const (
 	MPIImplementationOpenMPI MPIImplementation = "OpenMPI"
 	MPIImplementationIntel   MPIImplementation = "Intel"
 )
+
+const (
+	// JobSuspended means the job has been suspended.
+	JobSuspended common.JobConditionType = "Suspended"
+)
```

Comment:
Maybe we want to move `common.JobConditionType` to this repo, although we can follow up with other PRs.

Reply:
Yes, I was thinking about this. Still I would suggest in a follow up PR to separate the new logic and the refactoring.

Reply:
Sounds good.